releasedir/fs_blobs_dir.go

```diff
@@ -300,6 +304,23 @@ func (d FSBlobsDir) UploadBlobs() error {
 	return nil
 }
 
+func (d FSBlobsDir) containsSymlinks() bool {
+	files, err := d.fs.RecursiveGlob(filepath.Join(d.dirPath, "**/*"))
+	if err != nil {
+		return false
+	}
+
+	for _, file := range files {
+		fileInfo, _ := d.fs.Lstat(file)
```

Comment:
https://golang.org/pkg/os/#FileInfo is an interface. before using it, have to check for err.